fe/fe-core/src/main/java/org/apache/doris/system/SystemInfoService.java

```diff
@@ -901,7 +915,8 @@ public void clear() {
         this.idToReportVersionRef = null;
     }
 
-    public static Pair<String, Integer> validateHostAndPort(String hostPort) throws AnalysisException {
+    public static Triple<String, String, Integer> getIpHostAndPort(String hostPort, boolean strictCheck)
```

Comment:
why not name `strictCheck` to `isFqdnMode`?

Reply:
because when pod is deconstructed，the ip of pod may not be obtained，to prevent that we cannot drop backend